save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/DockerService.kt

```diff
@@ -39,6 +42,8 @@ class DockerService(
     private val agentRunner: AgentRunner,
     private val agentService: AgentService,
 ) {
+    private val areAgentsHaveStarted: ConcurrentMap<Long, AtomicBoolean> = ConcurrentHashMap()
```

Comment:
`haveAgentsStarted`? or diktat issue?

Reply:
Yep, but this variable has been here before, I just moved it